### PR TITLE
Fix documentation for xerr and yerr arguments in _axes.py

### PR DESCRIPTION
**Pull request by tropfcode**
## PR Summary
From issue #11156, documentation for xerr and yerr arguments when they are of shape(2,N) in the bar, barh, and errorbar methods suggests that '+' values are first row while '-' values are second row. The opposite is actually true -- documentation has been changed to more explicitly convey use of xerr and yerr.

## PR Checklist

- [ ] Has Pytest style unit tests
- [ X] Code is PEP 8 compliant
- [ ] New features are documented, with examples if plot related
- [ ] Documentation is sphinx and numpydoc compliant
- [ ] Added an entry to doc/users/next_whats_new/ if major new feature (follow instructions in README.rst there)
- [ ] Documented in doc/api/api_changes.rst if API changed in a backward-incompatible way

### CODE REVIEW

File: lib/matplotlib/axes/_axes.py [hunk 1]
```diff
@@ -1873,7 +1873,7 @@ def bar(self, x, height, width=0.8, bottom=None, *, align="center",
 
             - scalar: symmetric +/- values for all bars
             - shape(N,): symmetric +/- values for each bar
-            - shape(2,N): separate + and - values for each bar
+            - shape(2,N): First row lower errors, second row upper errors.
```
Comment (jklymak) [inline lib/matplotlib/axes/_axes.py]: Not a fan of the phrasing here, particularly given the line just before.  Also, the other entryes don't have a capital at the beginning.  

```
- shape(2,N): separate +/- values for each bar, the first row contains the - value.  
```

**Comment by jklymak** [comment]
#11161 was put in a few minutes later than yours, but has a more complete solution.  Would you be horribly offended if we closed this just to keep things simple?

**Comment by jklymak** [comment]
OK, thanks a lot for the help!  Usually we don't have such quick uptake on issues 😉

**Comment by tropfcode** [comment]
@jklymak No, please go with the best solution. It's what everyone wants and needs.

**Comment by Clear-up** [comment]
Oh I was still confused by this. I spent one hour to find the problem today. 

You only changed
`shape(2,N): Separate **- and +** values for each bar. First row`

BUT I just looked at the first two lines and thought they will be the same:
```
scalar: Symmetric **+/-** values for all data points.
shape(N,): Symmetric **+/-**values for each data point.
```
Why don't you change all the +/- into -/+ since it does not matter for the scalar and shape(N,) case?

**Comment by jklymak** [comment]
@Clear-up still not following what you were confused by.  This PR never got merged  Please open a new issue.

**Pull request closed without merging**